src/main/java/org/team199/robot2022/subsystems/Climber.java

```diff
@@ -19,44 +22,57 @@ public class Climber extends SubsystemBase {
     private static final double kNEOFreeSpeedRPM = 5680;
     private static final double kDiameterIn = 1;
     private static final double kDesiredRetractSpeedInps = 1;
-    private static final double kDesiredExtendSpeedInps = 24*.6;
-
-    // Torque is 2 * 9 * 0.5 = 9
-    // Torque on the motor is Torque / ( gearing = 9 ) = 1
-
-    private static final double kVoltsToCounterTorque = 10.5;
-
+    private static final double kDesiredExtendSpeedInps = 6;
```

Comment:
```suggestion
    private static final double kDesiredExtendSpeedInps = 24*.6;
```
This value should not be changed